thriftpy2/thrift.py

```diff
@@ -104,6 +104,7 @@ class TType(object):
     LIST = 15
     UTF8 = 16
     UTF16 = 17
+    BINARY = 18
```

Comment:
There was already a BINARY member at https://github.com/Thriftpy/thriftpy2/pull/139/files#diff-b55308698cb9187e83350298d1d941b56d1e956b1e36f36c00fe61e68145cef5R100

Reply:
Fixed in 6d1b8c9

Reply:
Does it affect other protocols like binary? Maybe we should check it in all protocols.

Reply:
Other protocols can not treat the new BINARY member if the value of it is not `11` anymore.

Reply:
Other protocols have been updated to support the  `BINARY` type